src/appsignal/config.py

```diff
@@ -134,12 +153,20 @@ def bool_to_env_str(value: Optional[bool]):
     return str(value).lower()
 
 
+def list_to_env_str(value: Optional[list[str]]):
+    if value is None:
+        return ""
```

Comment:
Note that, by returning `""`, this means that the private environment variable will be set to an empty string. I think we should avoid setting the corresponding private environment variable when the value is `None`.

There's already a check to avoid setting the environment variable if it is `None`: https://github.com/appsignal/appsignal-python/blob/b6228190ebbfc8e9659c1873a9146e4bf849ed36/src/appsignal/config.py#L207-L209

So I think this should return `None`:

```suggestion
    if value is None:
        return None
```

(If this change is done, it should probably also be done for `bool_to_env_str`)

Reply:
both updated 👍